package.json

```diff
@@ -24,8 +24,11 @@
     "wiki"
   ],
   "devDependencies": {
-    "eslint": "^9.12.0",
-    "@eslint/js": "^9.12.0"
+    "@eslint/compat": "^1.3.2",
+    "@eslint/js": "^9.35.0",
+    "eslint": "^9.35.0",
+    "eslint-plugin-es": "^4.1.0",
+    "globals": "^16.4.0"
```

Comment:
IMO that's not needed. It installs 3 more dependencies, which we do not control. NPM has a huge problem with "supply chain attacks" and we do not have the resources to stay on top of developer dependencies. 

See: https://vercel.com/blog/critical-npm-supply-chain-attack-response-september-8-2025
and: https://www.aikido.dev/blog/npm-debug-and-chalk-packages-compromised